homeassistant/components/coinmarketcap/sensor.py

```diff
@@ -54,9 +54,9 @@
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the CoinMarketCap sensor."""
-    currency_id = config.get(CONF_CURRENCY_ID)
-    display_currency = config.get(CONF_DISPLAY_CURRENCY).upper()
-    display_currency_decimals = config.get(CONF_DISPLAY_CURRENCY_DECIMALS)
+    currency_id = config[CONF_CURRENCY_ID]
+    display_currency = config[CONF_DISPLAY_CURRENCY].upper()
```

Comment:
Not related: The upper modification can be done in the config schema with `vol.Upper`.